hack/update-staging-godeps-dockerized.sh

```diff
@@ -87,6 +87,15 @@ function diffGodepManifest() {
   fi
 }
 
+# Create a fake git repo for staging to prevent godeps from complaining
+pushd "${KUBE_ROOT}/staging/src" >/dev/null
```

Comment:
Hmm.. the filter though is at the root though. I wonder if in some combination, this fake git would be copied out of the container (either via this or another script)